src/diffusers/loaders.py

```diff
@@ -1520,55 +1542,35 @@ def load_lora_into_text_encoder(
             if len(text_encoder_lora_state_dict) > 0:
                 logger.info(f"Loading {prefix}.")
                 rank = {}
+                text_encoder_lora_state_dict = convert_state_dict_to_diffusers(text_encoder_lora_state_dict)
```

Comment:
If it is in PEFT format, converting from PEFT to Diffusers Format and then back to PEFT Format is very redundant, WDYT?

Reply:
No this is how we want it. It's easier for slowly deprecating, to add new mappings and for maintenance/testing